Source/Runtime/Measurement Server.lvproj

```diff
@@ -141,6 +141,17 @@
 				<Item Name="niDMM Self-Test.vi" Type="VI" URL="/&lt;instrlib&gt;/niDMM/nidmm.llb/niDMM Self-Test.vi"/>
 				<Item Name="niDigital Self Test.vi" Type="VI" URL="/&lt;instrlib&gt;/niDigital/niDigital.llb/niDigital Self Test.vi"/>
 				<Item Name="niDCPower Self Test.vi" Type="VI" URL="/&lt;instrlib&gt;/niDCPower/nidcpower.llb/niDCPower Self Test.vi"/>
+				<Item Name="niDigital Get Session Reference.vi" Type="VI" URL="/&lt;instrlib&gt;/niDigital/niDigital.llb/niDigital Get Session Reference.vi"/>
```

Comment:
@jasonmreding @dixonjoel The changes in this file are always added whenever I open `Measurement Server.lvproj`. Therefore, I'm tempted to ask if we can check-in these changes.

Reply:
The LV project persists the dependencies of the project that come from outside of the project (e.g. vilib), which stinks for SCC. So if a particular library is missing (because you don't have the driver installed or what not), it tends to result in a change to the list of dependencies that get persisted in the project. I personally try to just not commit project file changes if I know I haven't actually made meaningful edits to the project in order to reduce PR noise. Otherwise, you end up playing Whack-A-Mole with the project file since what each developer has installed on their development machine is frequently different. For example, I don't have every single driver installed on my dev machine so the project will always have some missing dependencies when it loads. This means the project almost always loads with a dirty dot for me. With that said, I recognize this is a difficult thing to stay on top of so I've mostly learned to ignore the noise.

Reply:
Sure. I'll revert these changes then.